src/Aspire.Hosting/ResourceBuilderExtensions.cs

```diff
@@ -2299,26 +2299,29 @@ public static IResourceBuilder<T> WithComputeEnvironment<T>(this IResourceBuilde
     /// Adds support for debugging the resource in VS Code when running in an extension host.
     /// </summary>
     /// <param name="builder">The resource builder.</param>
+    /// <param name="resourceType">The type of resource, such as 'project'.</param>
     /// <param name="projectPath">The path to the project file.</param>
     /// <param name="debugAdapterId">The debug adapter ID to use. Ie, coreclr</param>
```

Comment:
Could the language-specific launch configuration have a default for the debug adapter?